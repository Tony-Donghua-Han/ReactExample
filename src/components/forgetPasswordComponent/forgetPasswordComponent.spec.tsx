import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import ForgetPasswordComponent from "./forgetPasswordComponent";

describe("ForgetPasswordComponent", (): void => {
    let wrapper: ShallowWrapper;
    beforeEach((): void => {
        wrapper = shallow(<ForgetPasswordComponent />);
    });

    test("should render without crushing", (): void => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
