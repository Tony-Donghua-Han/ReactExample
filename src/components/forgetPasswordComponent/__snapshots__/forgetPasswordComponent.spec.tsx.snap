// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgetPasswordComponent should render without crushing 1`] = `
"<div className=\\"forgetPasswordComponent\\" id=\\"forgetPasswordComponent\\">
  <div className=\\"title\\">
    Forget Password
  </div>
  <div className=\\"divider\\" />
  <div className=\\"form\\">
    <ForgetPasswordForm onSubmit={[Function: onSubmit]} />
  </div>
</div>"
`;
